### DISCUSSION TITLE
Tools

### DISCUSSION THREAD

**Discussion by rcvangemert** [pion/webrtc#1900] [General] [answered]
Hi, would be interested to know what tools or IDE are used to develop pion?

**Comment by Sean-Der** [comment, marked as answer]
Hi @rcvangemert 

I can't speak for other developers, but this is what I use

* vim with [vim-go](https://github.com/fatih/vim-go) my full [init.vim](https://github.com/Sean-Der/dot_files/blob/master/.config/nvim/init.vim)
* I run examples/`go test`/`golangci-lint run` from zsh
* I use wireshark and tcpdump a lot
* I have a local build of Chromium and make modifications to `third_party/webrtc` for debugging.

**Reply by rcvangemert** [reply]
thanks, i'll give vim a try.

**Reply by katepangLiu** [reply]
@rcvangemert 
Maybe vscode is a good option, I just use it to debug pion examples. 
@Sean-Der 
Can I commit some PRs about building pion/examples debuging-enviroment with vscode. 
It maybe stupid, but as a newbee like me, stupid quick start is necessary.

**Reply by Sean-Der** [reply]
I would love that @katepangLiu 

I just added you to the Pion organization. Feel free to add any pages to https://github.com/pion/webrtc/wiki.

I think `New to Go` and `How to develop Pion` articles could be really helpful.

**Comment by rcvangemert** [comment]
Thanks I have vim and vscode installed. leaning to vscode as it feels more
familiar!
